Packs/Elasticsearch/Integrations/Elasticsearch_v2/Pipfile

```diff
@@ -4,16 +4,34 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
-pytest = "*"
-pytest-mock = "*"
-requests-mock = "*"
-pytest-asyncio = "*"
+flake8 = "==4.0.1"
```

Comment:
this is just a copy of the pipfile from the dockerfiles repo so for other developers that would deal with this integration in the future it will be easier to create a virtual environment which is based on the elastic search integration dependancies, however its not a must for the PR.

Reply:
I have also added the dev dependancies in case one would need to run any unit-tests/flake8 stuff and so on. simply an add-on of the [dev-requirements-py3.txt](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt)